### ISSUE TITLE
Channels not loading after a a few hours

### ISSUE THREAD

**Issue by nfgb** [kiall/android-tvheadend#248]
Hi,

The app works great, however after a fews hours working, the channels don't load anymore. I've to restart the box in order to load again. There is no problem with TVHeadend server since it loads channels from other clients. The other sources in live channels app still work and the other apps in the box are ok.

**Comment by sotiris-bos** [comment]
> Hi,
> 
> The app works great, however after a fews hours working, the channels don't load anymore. I've to restart the box in order to load again. There is no problem with TVHeadend server since it loads channels from other clients. The other sources in live channels app still work and the other apps in the box are ok.

What kind of device are you using?

**Comment by nfgb** [comment]
wetek play 2. I end up abandoning this setup (installed another os). But I think the problem was related to the box itself.

**Comment by sotiris-bos** [comment]
> wetek play 2. I end up abandoning this setup (installed another os). But I think the problem was related to the box itself.

Yes there is a problem with AMLogic chips and Android's Exoplayer when playing back interlaced video that makes the boxes unresponsive after a few hours. I have this problem with the Xiaomi Mi Box and I am working on transcoding and deinterlacing the source video on the server as I cannot load a different OS on it.
